mods/persistence/controllers/subsystems/persistence.dm

```diff
@@ -199,9 +202,20 @@
 					var/turf/T = locate(x,y,z)
 					// This if statement, while complex, checks to see if we should save this turf.
 					// Turfs not saved become their default_turf after deserialization.
-					if(!istype(T) || istype(T, default_turf) || !T.should_save)
-						if(!istype(T) || !T.contents || !length(T.contents))
-							continue
+					if(!istype(T) || !LAZYLEN(T.contents))
+						continue
+					
+					//Ignore non-saved areas
+					var/area/TA = T.loc 
+					if(istype(TA) && (TA.area_flags & AREA_FLAG_IS_NOT_PERSISTENT))
+						for(var/mob/living/L in T)
```

Comment:
This should be added to the check in the loop which adds player minds to limbo if they're in an unsaved area for consistency, rather than being here.

Reply:
Speaking of that, does your mind stuff actually works? Because last time I checked, if you save while you're in a non saved area, you just cannot spawn on the menu. I told brawler about it, but I don't think he knows how to fix that.
But yeah, I can just remove that from there.